### PR TITLE
Two new components to center content and to add a field to a form

### PR DESCRIPTION
**Pull request by rumbado**

### CODE REVIEW

File: 15 LoginForm/src/pages/login/loginPage.tsx [hunk 1]
```diff
@@ -35,19 +36,13 @@ export class LoginPage extends React.Component<Props, State> {
...
-          </div>
-        </div>
-      </div>
+      <CenteredComponent>
+        <Header />
+        <Form loginInfo={this.state.loginInfo}
+          updateLoginInfo={this.updateLoginEntity.bind(this)}
+          performLogin={this.performLogin.bind(this)}
+          />
+      </CenteredComponent>
```
Comment (brauliodiez) [inline 15 LoginForm/src/pages/login/loginPage.tsx]: Should we name this CenteredContainer ?

File: 15 LoginForm/src/pages/login/form.tsx [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
+import {FormField} from '../common/formField';
 
 interface Props {
    loginInfo : LoginEntity;
    updateLoginInfo : (loginInfo : LoginEntity) => void;
    performLogin : () => void;
 }
 
-export const Form = (props: Props) => {
+export class Form extends React.Component<Props, {}> {
```
Comment (brauliodiez) [inline 15 LoginForm/src/pages/login/form.tsx]: Why the need of removing the stateless component?

File: 15 LoginForm/src/pages/common/centered.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import * as React from "react"
```
Comment (brauliodiez) [inline 15 LoginForm/src/pages/common/centered.tsx]: Missing in readme.md the containerComponent refactor (could be added as last step)

File: 15 LoginForm/src/pages/login/loginPage.tsx (hunk 1 above)
Comment (rumbado) [inline 15 LoginForm/src/pages/login/loginPage.tsx]: Yes, much more sense. I will rename it.

File: 15 LoginForm/src/pages/common/centered.tsx (hunk 3 above)
Comment (rumbado) [inline 15 LoginForm/src/pages/common/centered.tsx]: Forgot to update the readme. Sorry

File: 15 LoginForm/src/pages/login/form.tsx (hunk 2 above)
Comment (rumbado) [inline 15 LoginForm/src/pages/login/form.tsx]: I added the private function:
'''javascript
updateFieldValue(fieldName: string, fieldValue: any){
    const newLoginEntity = this.props.loginInfo;
    newLoginEntity[fieldName] = fieldValue;
    this.props.updateLoginInfo(newLoginEntity);
  }
''''
Can that be done without promoting it to class?

Comment (brauliodiez) [inline 15 LoginForm/src/pages/login/form.tsx]: Just declare it as a function, e.g. 

```javascript
updateFieldValue: (fielName : string, fieldValue: any) => {
//...
}
```
or

```javascript
function updateFieldValue: (fielName : string, fieldValue: any)  {
//...
}
```

**Comment by brauliodiez** [comment]
Cool stuff, Im going to merge it. @vicmator and Rocío can you close this issue and update the kanban?Thanks

**Pull request merged**